galaxychop/models.py

```diff
@@ -41,6 +42,18 @@ class Columns(enum.Enum):
 class GCDecomposeMixin:
     """Galaxy chop decompose mixin class."""
 
+    def get_clean_mask(self, X):
+        """Clean the Nan value of the circular parameter array."""
+        eps = X[:, Columns.circular_parameter.value]
+        (clean,) = np.where(~np.isnan(eps))
```

Comment:
check if  (clean,) = np.where(~np.isnan(eps)) is not the same of clean =~np.isnan(eps)